examples/gatsby-builder-example-no-plugin/.eslintrc

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
let's call the folder this lives in just `gatsby`

later we can update this as the canonical gatsby example the plugin when the plugin is good to go